libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/Spec/Conway.hs

```diff
@@ -22,6 +22,7 @@ spec = describe "Conway conformance tests" $ do
   xprop "ENACT" $ conformsToImpl @"ENACT" @ConwayFn @Conway
   prop "DELEG" $ conformsToImpl @"DELEG" @ConwayFn @Conway
   prop "POOL" $ conformsToImpl @"POOL" @ConwayFn @Conway
+  xprop "EPOCH" $ conformsToImpl @"EPOCH" @ConwayFn @Conway
```

Comment:
If the Specifications are all `TrueSpec`, is it necessary to have `xprop` here ?

Reply:
We need to keep this test pending _because_ the generators are all `TrueSpec`. It means that the generators are completely arbitrary, and the tests _fail_ because of that. The next phase, we focus on getting those specs constrained "just right".